2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_setup.py:_flush():69] setting env: {}
2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_init.py:_log_setup():336] Logging user logs to /home/ubuntuadm/trainseg/utils/wandb/run-20210415_142516-as9j1xkm/logs/debug.log
2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_init.py:_log_setup():337] Logging internal logs to /home/ubuntuadm/trainseg/utils/wandb/run-20210415_142516-as9j1xkm/logs/debug-internal.log
2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_init.py:init():369] calling init triggers
2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-15 14:25:16,399 INFO    MainThread:10654 [wandb_init.py:init():418] starting backend
2021-04-15 14:25:16,399 INFO    MainThread:10654 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-15 14:25:16,417 INFO    MainThread:10654 [backend.py:ensure_launched():123] starting backend process...
2021-04-15 14:25:16,426 INFO    MainThread:10654 [backend.py:ensure_launched():127] started backend process with pid: 10676
2021-04-15 14:25:16,429 INFO    MainThread:10654 [wandb_init.py:init():423] backend started and connected
2021-04-15 14:25:16,431 INFO    MainThread:10654 [wandb_init.py:init():465] updated telemetry
2021-04-15 14:25:16,441 INFO    MainThread:10654 [wandb_init.py:init():484] communicating current version
2021-04-15 14:25:17,548 INFO    MainThread:10654 [wandb_init.py:init():489] got version response 
2021-04-15 14:25:17,548 INFO    MainThread:10654 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-15 14:25:17,754 INFO    MainThread:10654 [wandb_init.py:init():522] starting run threads in backend
2021-04-15 14:25:19,489 INFO    MainThread:10654 [wandb_run.py:_console_start():1524] atexit reg
2021-04-15 14:25:19,489 INFO    MainThread:10654 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-15 14:25:19,489 INFO    MainThread:10654 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-15 14:25:19,505 INFO    MainThread:10654 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-15 14:25:19,505 INFO    MainThread:10654 [wandb_init.py:init():546] run started, returning control to user process
2021-04-15 14:27:43,405 INFO    MainThread:10654 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-15 14:27:43,407 INFO    MainThread:10654 [wandb_run.py:_restore():1466] restore
2021-04-15 14:27:45,668 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 813
  total_bytes: 813
}

2021-04-15 14:27:45,771 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 813
  total_bytes: 813
}

2021-04-15 14:27:46,215 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 813
  total_bytes: 3486
}

2021-04-15 14:27:46,539 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3486
  total_bytes: 3486
}

2021-04-15 14:27:46,641 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3486
  total_bytes: 3486
}

2021-04-15 14:27:46,743 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3486
  total_bytes: 3486
}

2021-04-15 14:27:46,845 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3486
  total_bytes: 3486
}

2021-04-15 14:27:46,947 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3486
  total_bytes: 3486
}

2021-04-15 14:27:47,049 INFO    MainThread:10654 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3486
  total_bytes: 3486
}

2021-04-15 14:27:48,495 INFO    MainThread:10654 [wandb_run.py:_show_summary():1771] rendering summary
2021-04-15 14:27:48,496 INFO    MainThread:10654 [wandb_run.py:_show_history():1809] rendering history
2021-04-15 14:27:48,497 INFO    MainThread:10654 [wandb_run.py:_show_files():1838] logging synced files
